screens/Home/AnnouncementScreen/AnnouncementScreen.tsx

```diff
@@ -71,7 +75,8 @@ const AnnouncementScreen = () => {
   }
 
   const renderItem = ({ item }: SectionListRenderItemInfo<Announcement>) => (
-    <TouchableOpacity >
+    <TouchableOpacity
+      onPress={() => {onAnnoucmentPress(item)}}>
       <AnnouncementCard {...item} />
     </TouchableOpacity>
   )
```

Comment:
Same in this screen overall. There cannot be mocking values everywhere and then hope for the best for the backend guys.

Reply:
I'm not doing this screen, I just had to modify it to allow the navigation to work. But don't worry we can do a PR to clean that when we have the database fetching function for announcements.